packages/frontpage/lib/data/user.ts

```diff
@@ -64,6 +64,19 @@ export const isBetaUser = cache(async () => {
   );
 });
 
+export const isAdmin = cache(async () => {
+  const user = await getUser();
+  if (!user) {
+    return false;
+  }
+
+  const isAdmin = await db.query.AdminUser.findFirst({
+    where: and(eq(schema.AdminUser.did, user.did)),
+  });
+
+  return isAdmin;
```

Comment:
A function called `isAdmin` not returning a boolean is unhinged.

```suggestion
  return Boolean(isAdmin);
```